examples/preprocessing/plot_target_encoder_cross_val.py

```diff
@@ -134,29 +150,42 @@
 model_no_cv = ridge.fit(X_train_no_cv_encoding, y_train)
 
 # %%
-# We evaluate the model on the non-cross validated encoding and see that it overfits:
+# We evaluate the model that did not use :term:`cross fitting` when encoding and
+# see that it overfits:
 print(
     "Model without CV on training set: ",
     model_no_cv.score(X_train_no_cv_encoding, y_train),
```

Comment:
Not sure if it would help to also rename the variables (e.f. `model_without_crossfitting` / `X_train_no_cf_encoding`).

As you wish.